packages/web-frontend/src/components/View/utils.js

```diff
@@ -40,7 +40,7 @@ const VIEW_TYPES = {
   ...SINGLE_VALUE_COMPONENTS,
 };
 
-export const getViewWrapper = ({ type, viewType }) => {
+export function getViewWrapper({ type, viewType }) {
```

Comment:
These functions were causing an error on the hot reload when changing view.js. I believe it is because of a circular dependency issue with hot reload. This fixes it because the functions get hoisted.